fdbserver/DataDistribution.actor.cpp

```diff
@@ -4147,7 +4147,7 @@ ACTOR Future<Void> ddSnapCreateCore(DistributorSnapRequest snapReq, Reference<As
 		std::vector<Future<Void>> disablePops;
 		for (const auto & tlog : tlogs) {
 			disablePops.push_back(
-				transformErrors(throwErrorOr(tlog.disablePopRequest.tryGetReply(TLogDisablePopRequest(snapReq.snapUID))), operation_failed())
+				transformErrors(throwErrorOr(tlog.disablePopRequest.tryGetReply(TLogDisablePopRequest(snapReq.snapUID))), snap_disable_tlog_pop_failed())
```

Comment:
Can the response have a more informative error code than snap_disable_tlog_pop_failed? If so unconditionally transforming it is actually losing important diagnostic information. Maybe at least log the original error?

Reply:
In many cases, original error message is just the operation succeeded or failed. Also, original error message is getting logged at TLog/storage/coordinator level.